index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="index.css" />
+  <title>Awesome Books</title>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700&display=swap"
+    rel="stylesheet" />
+  <script src="index.js" type="module" defer></script>
+  
+</head>
+
+<body>
+
+  <section id="nav-bar" class="nav-bar">
+    <div class="navbar">
+      <a href="#">
+        <h1>Awesome Books</h1>
+      </a>
+      <div class="main-links">
+        <ul>
+          <li><a id="listshow" href="#">List</a></li>
+          <li><a id="addshow" href="#">Add New</a></li>
+          <li><a id="contactshow" href="#">Contact</a> </li>
+        </ul>
+      </div>
+    </div>
+  </section>
+  <div id="date-time">fecha</div>
```

Comment:
- [ ] Displayed date is not properly formatted, and hence not displaying as intended, kindly ensure to fix it:

**Incorrect date format:**
![image](https://user-images.githubusercontent.com/12745474/213084840-806ab79b-8a84-4270-a721-59bc1f238983.png)